doc/level1.rst

```diff
@@ -231,18 +233,20 @@ to an OpenPGP backend for handling).
 Header injection in outbound mail
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-During message composition, if the ``From:`` header of the
-outgoing e-mail (the ``sender address``) matches an address for which the Autocrypt-capable MUA
-has secret key material (``own_state.secret_key``), it
-SHOULD include an Autocrypt header. This header MUST contain the
-corresponding public key material (``own_state.public_key``) as the ``keydata``
-attribute, and the ``sender address`` as the ``addr`` attribute.  The most
-minimal Level 1 compliant MUA will only include these two attributes.  If
-``own_state.prefer_encrypt`` is set to ``mutual``, then the header MUST
-have a ``prefer-encrypt`` attribute with the value ``mutual``.
+During message composition, if the ``From:`` header of the outgoing
+e-mail (the ``from-addr``) matches an address for which the
+Autocrypt-capable MUA has secret key material
+(``accounts[from-addr].secret_key``), it SHOULD include an Autocrypt
```

Comment:
the "it" here sounds like it might refer to the secret key material, I suggest making it an explicit reference to the message